README.md

```diff
@@ -10,3 +10,4 @@ For example:
 * [Elad Shechter](https://elad.medium.com/): CSS. Writing language: en.
 * [David Meir-Levy](https://www.davidlevy.co.il): Front end. Writing language: en.
 * [Uri Shaked](https://urish.org): IoT, Electronics, Frontend and more. English.
+* [Yakov Rakhamimov](https://yakov.dev): Frontend development & soft skills. Writing language: en.
```

Comment:
Please change to ". S".

Reply:
Hey, not sure what you mean.

Reply:
@jacobra19
I mean to change the " & s" to ", S".
I created this specific flow, so in the future I could create a real website from this list and the comma separator will help me create a tag management and search. How ever, in the mean time this readme file will still be useful to everyone.

Reply:
ok no problem, thanks for the reply!

Reply:
@jacobra
Hi, I wont to merge you pr but you have conflicts. Please solve them.